libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -1566,5 +1566,35 @@
         "value": "Use two fingers to move the map",
         "comment": "This message will appear when moving map on GeoChart is disabled by one finger and available only with two.",
         "limit": 0
+    },
+    "sorting.disabled.explanation.attribute|insight": {
+        "value": "Sorting is not possible for this insight configuration.",
+        "comment": "",
```

Comment:
All the texts should have some context comment for translators.

Reply:
fixed